samples/msal-node-samples/ElectronSystemBrowserApp/README.md

```diff
@@ -0,0 +1,204 @@
+# MSAL Node Electron Sample Using System browser and Custom URL Scheme
+
+An Electron application built with TypeScript that uses the MSAL Node library to acquire and store access tokens to authenticate with the Microsoft Graph API.
+
+## Should I use this sample
+
+Understanding the way MSAL Node is used in this sample will help you if you're interested in building an Electron.js application to authenticate and acquire tokens using the system browser and custom URL scheme.
```

Comment:
Can add a few more lines here? What advantage does using the system browser provide?